### ISSUE TITLE
Co-expression: should we overlap with mutaiton data or CNAs?

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#4006]
Currently, we are overlaying mutational data on top. I couldn't recall why we did that. Would it make more sense to overlay copy number data since CN may have a greater impact on expression?

![image](https://user-images.githubusercontent.com/840895/37679410-32184562-2c57-11e8-9205-277fd32f43b2.png)

**Comment by adamabeshouse** [comment]
One option would be to do both: have checkboxes for Show Mutations and Show CNA, and if both are checked then color if either mutated or CNA, "KRAS altered" etc

**Comment by tmazor** [comment]
Funny you should bring this up - I had a similar thought earlier today when looking at this page. I think we should have options for both. I had been thinking of a toggle like we have on the plots tab, but checkboxes like @adamabeshouse suggested would be even better. Or even better, we could also have an option to define a sample as altered according to the OQL used in the query.

**Comment by schultzn** [comment]
Agree.. both would be good.. copy-number, makes a lot of sense (and I often wondered why we don’t have it), but we should keep mutations.

**Comment by jjgao** [comment]
For copy number, should we just use "altered or not"? Or differentiate between -2 and 2? How about -1 and 1?

**Comment by adamabeshouse** [comment]
@jjgao also what to consider altered if both mutation and CNA selected?

**Comment by schultzn** [comment]
I feel like we used to have this functionality in an old version of the Plots tab?

We could try to versions:
1 - Change the outline of each circle to indicate the copy-number change (red, pink, grey, light blue, dark blue) and fill in the circle as we do now to indicate mutation status
2 - Flip the above around and use the circle color for copy-number, and color the outline of the circle differently to indicate mutations.

What do people think?

**Comment by adamabeshouse** [comment]
@schultzn as-is I don't think the border will be visible enough for that, but maybe if we thickened it

**Comment by schultzn** [comment]
Probably.. can you play around with it a bit?

**Comment by adamabeshouse** [comment]
yes, I will post screenshots

**Comment by adamabeshouse** [comment]
it just occurred to me this only works in plots tab when the gene is the same, because otherwise which CNA do you show? in plots tab we only show CNA when the gene is the same.

**Comment by adamabeshouse** [comment]
@schultzn and I discussed and we're going to put this off
